WordImpress/Give
# Issue/1473

## Description
This PR is for #1473

## How Has This Been Tested?
Tested this with different scenarios:
- changing form settings to single and multiple forms.
- changing form settings to single and multiple forms after saving form settings.
- set custom amount as default settings  to `enabled` and then check the form in front end. 
- set custom amount as default settings to `disabled` and then check the form in front end.

## Screenshots (jpeg or gifs if applicable):
**Frontend Workability**
![gif_about_functioning](http://g.recordit.co/jRXSTlvURU.gif)

**Form Settings in admin**
![image](https://user-images.githubusercontent.com/1852711/28264611-56ec6810-6b09-11e7-95a9-4ba93b53b885.png)

## Types of changes
New feature (non-breaking change which adds functionality)=

## Checklist:
